requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
+element_interface==0.5.0
+ipywidgets==8.0.4
+matplotlib==3.7.0
```

Comment:
```suggestion
```

Reply:
Since `matplotlib` is a dependency of `datajoint-python`, I am inclined to remove this version pin from here.

Reply:
Okay ok Sir